apps/api_web/test/api_web/router_test.exs

```diff
@@ -43,4 +43,20 @@ defmodule ApiWeb.RouterTest do
                |> authenticated_accepts(["event-stream"])
     end
   end
+
+  describe "CORS" do
+    @endpoint ApiWeb.Endpoint
+
+    test "returns the appropriate headers when an OPTIONS request is made", %{conn: conn} do
+      conn =
+        conn
+        |> put_req_header("x-api-key", conn.assigns.api_key)
+        |> put_req_header("access-control-request-headers", "x-api-key")
+        |> put_req_header("access-control-request-method", "GET")
+        |> put_req_header("origin", "http://localhost/")
+
+      response = options(conn, "/_health")
+      assert response.status == 200
```

Comment:
The test name is "returns the appropriate headers when an OPTIONS request is made" but the only thing that's being asserted is the status. Did you mean to check the response headers, too?

Reply:
Fixed.